examples/ensemble/plot_gradient_boosting_quantile.py

```diff
@@ -11,6 +11,10 @@
 import matplotlib.pyplot as plt
 
 from sklearn.ensemble import GradientBoostingRegressor
+from sklearn.metrics import pinball_loss, mean_squared_error
+from sklearn.model_selection import train_test_split
+
+from pandas import DataFrame
```

Comment:
the line 

```
#----------------------------------------------------------------------
```

below should be replace by a cell marker (`# %%`).

You can check the rendering of the example by clicking the link named `ci/circleci: doc artifact` in the CI report of the PR.